<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VCMainWindow</class>
 <widget class="QMainWindow" name="VCMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Volume Cartographer</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::DockOption::AllowNestedDocks|QMainWindow::DockOption::AllowTabbedDocks|QMainWindow::DockOption::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="central" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="toolButtonLayout">
         <item>
          <widget class="QPushButton" name="btnSegTool">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Segmentation Tool</string>
           </property>
           <property name="icon">
            <iconset theme="format-justify-fill"/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPenTool">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Pen Tool</string>
           </property>
           <property name="icon">
            <iconset theme="edit-find-replace"/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="viewerLayout">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tabSegment">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <attribute name="icon">
             <iconset theme="image-x-generic"/>
            </attribute>
            <attribute name="title">
             <string>Segment</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetVolumes">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>64</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFeature::DockWidgetClosable|QDockWidget::DockWidgetFeature::DockWidgetFloatable|QDockWidget::DockWidgetFeature::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::DockWidgetArea::LeftDockWidgetArea|Qt::DockWidgetArea::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Volume Package</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetVolumesContent">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="vLayoutMenu">
       <item>
        <widget class="QGroupBox" name="grpVolManager">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="lblVpkgName">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>VOLUME_PACKAGE_NAME</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="treeWidgetSurfaces">
            <property name="indentation">
             <number>0</number>
            </property>
            <property name="columnCount">
             <number>1</number>
            </property>
            <attribute name="headerVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="headerMinimumSectionSize">
             <number>50</number>
            </attribute>
            <attribute name="headerDefaultSectionSize">
             <number>120</number>
            </attribute>
            <attribute name="headerStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Surface ID</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbFilterSegs">
            <item>
             <property name="text">
              <string>show all</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>filter by focus point</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_vol">
            <property name="text">
             <string>Volume</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="volSelect"/>
          </item>
          <item>
           <widget class="QPushButton" name="assignVol">
            <property name="text">
             <string>Assign Segmentation to Volume</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetOpSettings">
   <property name="minimumSize">
    <size>
     <width>103</width>
     <height>143</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFeature::DockWidgetClosable|QDockWidget::DockWidgetFeature::DockWidgetFloatable|QDockWidget::DockWidgetFeature::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::DockWidgetArea::LeftDockWidgetArea|Qt::DockWidgetArea::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Layer Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="opsSettingsContent">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_3"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetSegmentation">
   <property name="minimumSize">
    <size>
     <width>352</width>
     <height>548</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::DockWidgetArea::LeftDockWidgetArea|Qt::DockWidgetArea::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Segmentation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetSegmentationContent">
    <layout class="QVBoxLayout" name="verticalLayout_14">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QGroupBox" name="grpSeg">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="segMethodLabel">
          <property name="text">
           <string>Method</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbSegMethods"/>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContent">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>367</width>
             <height>360</height>
            </rect>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QStackedWidget" name="segParamsStack">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <widget class="QWidget" name="lrpsParams">
               <layout class="QHBoxLayout" name="lrpsParamsLayout">
                <item>
                 <layout class="QFormLayout" name="formLayout">
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Alpha</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" colspan="2">
                   <widget class="QLineEdit" name="edtDistanceWeightVal"/>
                  </item>
                  <item row="0" column="0" colspan="2">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Distance Weight</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="edtAlphaVal"/>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="edtDeltaVal"/>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Delta</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="0">
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>K2</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QLineEdit" name="edtK2Val"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QFormLayout" name="formLayout_2">
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Beta</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" colspan="2">
                   <widget class="QSpinBox" name="edtWindowWidthVal"/>
                  </item>
                  <item row="0" column="0" colspan="2">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Maxima Window Width</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="edtBetaVal"/>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="edtK1Val"/>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>K1</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="0" colspan="2">
                   <widget class="QCheckBox" name="includeMiddleOpt">
                    <property name="text">
                     <string>Consider Previous</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Targets</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="startEndLayout" stretch="1,1">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QRadioButton" name="radioBackwardSlice">
              <property name="text">
               <string>Backward Slice</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupBackward</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBackwardSlice">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioBackwardAnchor">
              <property name="text">
               <string>Backward Anchor</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupBackward</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioBackwardNoRun">
              <property name="text">
               <string>None</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupBackward</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QRadioButton" name="radioForwardSlice">
              <property name="text">
               <string>Forward Slice</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupForward</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinForwardSlice"/>
            </item>
            <item>
             <widget class="QRadioButton" name="radioForwardAnchor">
              <property name="text">
               <string>Forward Anchor</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupForward</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioForwardNoRun">
              <property name="text">
               <string>None</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupForward</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnStartSeg">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpEditing">
       <property name="title">
        <string>Editing</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Impact Range</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sldImpactRange">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="value">
             <number>8</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labImpactRange">
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnEvenlySpacePoints">
          <property name="text">
           <string>Evenly Space Points</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetLocation">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetLocationContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>82</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>focus</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="dspNY">
         <property name="minimum">
          <double>-10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="dspNX">
         <property name="minimum">
          <double>-10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>normal</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="dspNZ">
         <property name="minimum">
          <double>-10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="sliceX">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="sliceY">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="sliceZ">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetOpList">
   <property name="windowTitle">
    <string>Layers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="opsListContent">
    <layout class="QGridLayout" name="gridLayout_4"/>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <zorder>dockWidgetOpList</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupForward"/>
  <buttongroup name="buttonGroupBackward"/>
 </buttongroups>
</ui>
